context/index.js

```diff
@@ -0,0 +1,89 @@
+let contexts = new Map()
+
+let _createContext = name => {
```

Comment:
`_` is too encrypting for the place where minifier will change names.

What about `buildContext` and `createContext` (`build` is about building the variable and `create` is about side effects)?